terraform/environments/preview/s3_buckets.tf

```diff
@@ -212,3 +212,178 @@ resource "aws_s3_bucket" "submissions_bucket" {
     target_prefix = "digitalmarketplace-submissions-preview-preview/"
   }
 }
+
+# Cross Region replication buckets
+resource "aws_s3_bucket" "cross_region_documents_s3_bucket" {
+  provider = "aws.london"
+  bucket   = "digitalmarketplace-cross-region-documents-preview-preview"
+  acl      = "private"
+  region   = "eu-west-2"
+
+  versioning {
+    enabled = true
+  }
+
+  }
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Principal": {"AWS": "arn:aws:iam::${var.aws_development_account_id}:role/replication"},
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucket",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": "arn:aws:s3:::digitalmarketplace-cross-region-documents-preview-preview"
+    },
+  ]
+}
+POLICY
+
+  replication_configuration {
```

Comment:
The replication configuration needs to live on the bucket you're replicating from, the source, rather than the destination.

In the backups terraform I defined the destination bucket above the source bucket because semantically it makes sense to have the  replication bucket exist before you can tell something to replicate in to it.

Reply:
I've pushed a couple of commits which:
- move the `replication_configuration` blocks to the source buckets
- get rid of the destination bucket policies
- fixes trailing commas on the IAM role lists

I ran `make plan` for the roles in dev/prod and all seems fine; I can't run the plan for the buckets until those roles are created for real, as they're on different Makefiles.